README.md

```diff
@@ -67,43 +67,10 @@ BUILDKITE_ANALYTICS_TOKEN=xyz rake
 
 Add the `BUILDKITE_ANALYTICS_TOKEN` secret to your CI, push your changes to a branch, and open a pull request 🎉
 
-```bash
-git checkout -b add-buildkite-test-engine
-git commit -am "Add Buildkite Test Engine"
-git push origin add-buildkite-test-engine
-```
-
-### VCR
-If your test suites use [VCR](https://github.com/vcr/vcr) to stub network requests, you'll need to modify the config to allow actual network requests to Test Engine.
-
-```
-VCR.configure do |c|
-  c.ignore_hosts "analytics-api.buildkite.com"
-end
-```
```

Comment:
Will we add some documentation about VCR to the docs site?

Reply:
Good catch! I raised another PR to the doc site (can't post link here).